tests/ml/sklearn_test.py

```diff
@@ -188,6 +189,34 @@ def test_sklearn_incremental_predictor():
     np.testing.assert_array_almost_equal(pred_in_memory, df_test.pred.values, decimal=1)
 
 
+def test_sklearn_incremental_predictor_classification():
+    df = vaex.ml.datasets.load_iris_1e5()
+    df_train, df_test = df.ml.train_test_split(test_size=0.1, verbose=False)
+
+    features = df_train.column_names[:4]
+    target = 'class_'
+
+    incremental = IncrementalPredictor(model=SGDClassifier(learning_rate='constant', eta0=0.01),
+                                       features=features,
+                                       batch_size=10_000,
+                                       num_epochs=3,
+                                       shuffle=False,
+                                       prediction_name='pred')
+
+    incremental.fit(df=df_train, target=target, classes=[0, 1, 2])
+    df_train = incremental.transform(df_train)
+
+    # State transfer
+    state = df_train.state_get()
+    df_test.state_set(state)
+
+    assert df_test.column_count() == 6
+    assert df_test.pred.values.shape == (10050,)
```

Comment:
I don't like magic values :)
```suggestion
    assert df_test.pred.values.shape == (len(df),)
```